ci-operator/step-registry/baremetal/lab/agent/conf/static/baremetal-lab-agent-conf-static-commands.sh

```diff
@@ -29,6 +29,11 @@ EOF
 for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   # shellcheck disable=SC1090
   . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  if [[ "${name}" == *-a-* ]] && [ "${ADDITIONAL_WORKERS_DAY2}" == "true" ]; then
```

Comment:
@zniu1011 can you double check the other steps and synchronize if any similar loop is left such that we have them all in the same page? Even if no test configs are defined for multiarch compute nodes, we can just verify the change against regressions.

Reply:
Yes, I had run the regressions, it will not effected for single arch cases.

Reply:
@zniu1011 there are more instances of blocks that look through the hosts where we want to provision the ABI cluster. We should update them too while working on this enablement.

- https://github.com/openshift/release/blob/04545ae805a262595b0b1c5c0a78bd8f5e44936e/ci-operator/step-registry/baremetal/lab/agent/conf/dynamic/baremetal-lab-agent-conf-dynamic-commands.sh#L30
- https://github.com/openshift/release/blob/343ccf9d219d62337650e25394363a5eeb959686/ci-operator/step-registry/baremetal/lab/agent/install/ephemeral/boot-unconfigured-image/baremetal-lab-agent-install-ephemeral-boot-unconfigured-image-commands.sh#L53-L67
- https://github.com/openshift/release/blob/04545ae805a262595b0b1c5c0a78bd8f5e44936e/ci-operator/step-registry/baremetal/lab/agent/install/ephemeral/gen-ignition/baremetal-lab-agent-install-ephemeral-gen-ignition-commands.sh#L95-L106

Reply:
@aleskandro These new added if blocks are intended to skip the day2 nodes when configuring node network, mounting iso and powering nodes, to make sure the control nodes can install properly in agent install without day2 nodes, which reserved in the first pre steps too. Since we only run the dynamic flow for the day2 cases, the other changes will not being used for now or for ever.

Reply:
Thanks @zniu1011 ... We don't use them now, but for example, if we want to do ephemeral installations later, the only change required to allow day2 scale up is probably just that. So it's worth it to do that change wherever it could be required as part of a single PR that targets a full enablement of the current agent steps to support multiarch compute nodes at day2.